src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const { getUserById } = require('../services/users.service');
+const {
+  addExpense,
+  deleteExpense,
+  getExpense,
+  getExpenses,
+  updateExpenseById,
+} = require('../services/expenses.service');
+
+const getAllExpenses = (req, res) => {
+  const { userId, from, to, categories } = req.query;
+  let preparedExpenses = getExpenses();
+
+  if (userId) {
+    preparedExpenses = preparedExpenses
+      .filter(expense => expense.userId === +userId);
```

Comment:
```suggestion
      .filter(expense => expense.userId === Number(userId));
```

The same in other places